package.json

```diff
@@ -17,7 +17,7 @@
     "postinstall": "simple-git-hooks",
     "format": "prettier --write --cache .",
     "lint": "eslint --cache .",
-    "typecheck": "tsc -p scripts --noEmit && tsc -p playground --noEmit",
+    "typecheck": "tsc -p scripts --noEmit && tsc -p playground --noEmit && tsc -p packages/create-vite --noEmit && tsc -p packages/vite --noEmit",
```

Comment:
```
pnpm -r --parallel run typecheck
```
Maybe we could run in parallel?

Reply:
I don't know if it's worth creating a workspace for `scripts` for that. Did use `pnpm --parallel` for the three others